### PR TITLE
US4: Submit slots

### PR DESCRIPTION
**Pull request by drauf**

### CODE REVIEW

**Comment by r-kruk-kainos** [comment]
Code review done, everything seems ok

**Comment by drauf** [comment]
It's done, but should be merged after US-2.

File: src/main/java/resources/tdpRecruitmentSlotsResource.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.Date;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.List;
+
+@Path("/slots")
+@Produces(MediaType.APPLICATION_JSON)
+public class tdpRecruitmentSlotsResource {
```
Comment (wojciechh-kainos) [inline src/main/java/resources/tdpRecruitmentSlotsResource.java]: Java classes start with a capital letter

File: src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                    <input type="checkbox" ng-model="slot.available">
+                    <p ng-show="slot.available">AVAILABLE</p>
+                    <p ng-hide="slot.available">BUSY</p>
+                </td>
+            </tr>
+        </tbody>
+    </table>
+    <button type="button" ng-click="updateSlots()">SUBMIT SLOTS</button>
+
+    <style>
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html:30]: This style should be moved to a separate file - one team already added a main.css file, move it when the file is merged

**Pull request merged**